stockflux-securities-master/src/App.js

```diff
@@ -8,11 +8,11 @@ import { Redirect, Route, Switch } from "react-router-dom";
 import "./App.css";
 
 const App = () => {
-
   return (
     <div>
       <Components.Titlebar />
       <Switch>
+        <Route exact path="/table/:securityId" component={SecuritiesTable} />
```

Comment:
Why is there a route for the table with a `securityId`?

Reply:
I have removed this now and used the history prop to route from the form to the table if the deletion is successful. If it's unsuccessful then it will stay on the form and display an error. 

As we discussed, there is no need for a message after redirecting so no prop is being passed down anymore